test/jit.cpp

```diff
@@ -53,9 +53,11 @@ TEST(JIT, CPP_JIT_HASH) {
 
     // Creating a kernel
     {
-        array d    = a + b;
-        array e    = a + c;
-        array f1   = d * e - e;
+        array d  = a + b;
+        array e  = a + c;
+        array f1 = d * e - e;
+
+        f1.eval();
```

Comment:
why is eval needed here ?

Reply:
It was left over from debugging. Removed.